packages/react-reconciler/src/__tests__/__snapshots__/ReactIncrementalPerf-test.js.snap

```diff
@@ -6,10 +6,11 @@ exports[`ReactDebugFiberPerf captures all lifecycles 1`] = `
   ⚛ AllLifecycles [mount]
     ⚛ AllLifecycles.componentWillMount
     ⚛ AllLifecycles.getChildContext
-  ⚛ (Committing Changes)
-    ⚛ (Committing Host Effects: 1 Total)
-    ⚛ (Calling Lifecycle Methods: 1 Total)
-      ⚛ AllLifecycles.componentDidMount
+
```

Comment:
AFAICT the changes here are only because the commit phase is now a separate entry point. Other than that it's the same.